pkg/process/checks/process.go

```diff
@@ -76,9 +80,12 @@ func (p *ProcessCheck) Init(_ *config.AgentConfig, info *model.SystemInfo) {
 		log.Infof("no network ID detected: %s", err)
 	}
 	p.networkID = networkID
-
 	p.maxBatchSize = getMaxBatchSize()
 	p.maxBatchBytes = getMaxBatchBytes()
+
+	p.checkCount = 0
```

Comment:
Nit: This should be 0 by default. No need to initialize it.

Reply:
Done